fuzzers/binary_only/qemu_launcher/src/client.rs

```diff
@@ -75,7 +74,7 @@ impl Client<'_> {
         }
 
         #[cfg(not(feature = "injections"))]
-        let injection_module = None;
+        let injection_module = Option::<InjectionModule>::None;
```

Comment:
@tokatoka also due to the generics change?

Reply:
no this is unrelated